lib/rock_rms/resources/person.rb

```diff
@@ -35,16 +35,21 @@ def find_person_by_name(full_name, options = {})
         )
       end
 
-      def create_person(first_name:, last_name:, email:, connection_status_value_id: nil, record_status_value_id: nil)
+      def create_person(first_name:, last_name:, email:, connection_status_value_id: nil, record_status_value_id: nil, record_type_value_id: 1)
         options = {
           'IsSystem' => false,
           'FirstName' => first_name,
           'LastName' => last_name,
           'Email' => email,
           'Gender' => 1,
           'ConnectionStatusValueId' => connection_status_value_id,
-          'RecordStatusValueId'     => record_status_value_id
+          'RecordStatusValueId'     => record_status_value_id,
+          'RecordTypeValueId'     => record_type_value_id,
```

Comment:
What happens now?

If `RecordTypeValueId` is not passed in the API, does the person record get created with the value of 1 by default? Or is it `null`?

Reply:
As now (not sending the RecordTypeValueId key), once the record is created in the Rock DB, it gets assigned a RecordTypeValueId of 1.